environment-model/environment-model_shacl.ttl

```diff
@@ -3,134 +3,286 @@
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-environment-model:Asset_Shape a sh:NodeShape ;
-    sh:property [ skos:example "0.1 m" ;
-            sh:datatype xsd:float ;
-            sh:description "real texture resolution in meter (max?)"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of texture_resolution failed!"@en ;
-            sh:minCount "0"^^xsd:unsignedInt ;
-            sh:name "texture_resolution"@en ;
-            sh:path environment-model:environment-model_quality_texture_resolution ],
-        [ skos:example "Trian3DBuilder, version 7.7" ;
-            sh:datatype xsd:string ;
-            sh:description "tool for the creation of the data"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of creation_version failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
-            sh:name "creation_version"@en ;
-            sh:path environment-model:environment-model_project_creation_version ],
-        [ skos:example "high" ;
+environment-model:EnvironmentModelShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environment-model:QuantityShape ;
+            sh:order 2 ;
+            sh:path environment-model:quantity ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environment-model:QualityShape ;
+            sh:order 3 ;
+            sh:path environment-model:quality ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environment-model:ProjectShape ;
+            sh:order 0 ;
+            sh:path environment-model:project ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environment-model:FormatShape ;
+            sh:order 1 ;
+            sh:path environment-model:format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environment-model:GeoreferenceShape ;
+            sh:order 7 ;
+            sh:path environment-model:georeference ] ;
+    sh:targetClass environment-model:EnvironmentModel .
+
+    environment-model:GeoreferenceShape a sh:NodeShape ;
```

Comment:
Is this another GeoreferenceShape then the one which is developed here?
I guess it is the same. We should link it here and not duplicate it.

The _instance.json file then also needs an update.

Reply:
outdated? see branch environment-model and 
https://github.com/GAIA-X4PLC-AAD/ontology-management-base/pull/44

Reply:
unresolved, still the same in the code.